src/rosdep2/platforms/pip.py

```diff
@@ -29,8 +29,8 @@
 
 from __future__ import print_function
 
+import importlib.metadata
```

Comment:
The `importlib.metadata` module is only present in Python 3.8 and newer. This package must support Python 3 versions as far back as 3.6 (for RHEL 8).

The suggested pattern here is used in various other packages to prefer the Python 3.8+ module if present, and fall back to the backport package if it can't be found.

```suggestion
try:
    from importlib.metadata import version
except ImportError:
    from importlib_metadata import version
```

You'll also need to add this to the `install_requires`:   `importlib-metadata; python_version < "3.8"`

...and update the implementation to simply call `version` instead of `importlib.metadata.version`.